src/platform/Darwin/BlePlatformDelegateImpl.mm

```diff
@@ -103,16 +103,13 @@
         uint16_t BlePlatformDelegateImpl::GetMTU(BLE_CONNECTION_OBJECT connObj) const { return 0; }
 
         bool BlePlatformDelegateImpl::SendIndication(
-            BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId, const ChipBleUUID * charId, PacketBuffer * pBuf)
+            BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId, const ChipBleUUID * charId, PacketBufferHandle pBuf)
         {
-            if (pBuf) {
-                PacketBuffer::Free(pBuf);
-            }
             return false;
         }
 
         bool BlePlatformDelegateImpl::SendWriteRequest(
-            BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId, const ChipBleUUID * charId, PacketBuffer * pBuf)
+            BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId, const ChipBleUUID * charId, PacketBufferHandle pBuf)
```

Comment:
There are some `NULL != pBuf` bits in this function that need to become `!pBuf.IsNull()`